test/signed-log.spec.js

```diff
@@ -25,14 +30,22 @@ Object.keys(testAPIs).forEach((IPFS) => {
       repo: config.defaultIpfsConfig.repo + '-log-signed' + new Date().getTime()
     })
 
+    let identityStore, signingStore
+
     before(async () => {
       rmrf.sync(ipfsConfig.repo)
       rmrf.sync(identityKeysPath)
       rmrf.sync(signingKeysPath)
       await fs.copy(identityKeyFixtures, identityKeysPath)
       await fs.copy(signingKeyFixtures, signingKeysPath)
-      testIdentity = await IdentityProvider.createIdentity({ id: 'userA', identityKeysPath, signingKeysPath })
-      testIdentity2 = await IdentityProvider.createIdentity({ id: 'userB', identityKeysPath, signingKeysPath })
+
+      identityStore = await storage.createStore(identityKeysPath)
+      signingStore = await storage.createStore(signingKeysPath)
+      const keystore = new Keystore(identityStore)
+      const signingKeystore = new Keystore(signingStore)
```

Comment:
Could this functionality (creating the storage/store) be moved to IdentityProvider, so that it creates those by default? I feel this adds extra hoops for users who wish to use IdentityProvider to create an identity (note: one line) and making it as simple as possible (one line) for the user provides a better UX. What do you think?